src/js/bootstrap-slider.js

```diff
@@ -1148,6 +1148,9 @@ const windowIsDefined = (typeof window === "object");
 
 				this.handle1.style[this.stylePos] = positionPercentages[0]+'%';
 				this.handle1.setAttribute('aria-valuenow', this._state.value[0]);
+				if (isNaN(this.options.formatter(this._state.value[0])) ) {
+					this.handle1.setAttribute('aria-valuetext', this.options.formatter(this._state.value[0]));
```

Comment:
@mediaformat we are also setting `aria-valuenow` on `this.handle2` below, so I think it would be valuable to set it on handle2 as well